app/uk/gov/hmrc/tai/connectors/TaxAccountConnector.scala

```diff
@@ -69,8 +75,8 @@ class TaxAccountConnector @Inject()(
       val requestHeader = headersForUpdate(hc, version, sessionOrUUID, desConfig.originatorId)
 
       httpHandler
-        .postToApi[List[IabdUpdateAmount]](url, amountList, APITypes.DesIabdUpdateEstPayAutoAPI)(
-          requestHeader,
+        .postToApi[List[IabdUpdateAmount]](url, amountList, APITypes.DesIabdUpdateEstPayAutoAPI, requestHeader)(
+          hc,
```

Comment:
can remove implicits here

Reply:
I need to state `IabdUpdateAmountFormats.formatList`, so I left that in and changed hc to implicitly